test/data/marriage-abroad-questions-and-responses.yml

```diff
@@ -1,99 +1,163 @@
 ---
 :country_of_ceremony?:
+- afghanistan
```

Comment:
I'm not sure if these countries should have been added in but was the result of running the regression tests.